Sources/Moya/Plugins/AccessTokenPlugin.swift

```diff
@@ -28,15 +37,11 @@ public struct AccessTokenPlugin: PluginType {
     /// The access token to be applied in the header.
     public let token: String
 
-    private var authVal: String {
-        return "Bearer " + token
-    }
-
     /**
      Initialize a new `AccessTokenPlugin`.
 
      - parameters:
-       - token: The token to be applied in the pattern `Authorization: Bearer <token>`
+       - token: The token to be applied in the pattern `Authorization: <AuthorizationType> <token>`
     */
     public init(token: String) {
```

Comment:
We probably need to since it will be used in later stages of the endpoint -> request cycle.